### ISSUE TITLE
bug: enable_ephemeral_storage broken by update to support EFS to persist Atlantis state

### ISSUE THREAD

**Issue by MarkIannucci** [terraform-aws-modules/terraform-aws-atlantis#254]
@MarkIannucci 
first of all, thank you! 🙏 

Unfortunately this is not working if I want to have `enable_ephemeral_storage=true`

```
╷
│ Error: Invalid index
│ 
│   on .terraform/modules/atlantis/main.tf line 438, in resource "aws_efs_mount_target" "this":
│  438:   file_system_id  = aws_efs_file_system.this[0].id
│     ├────────────────
│     │ aws_efs_file_system.this is empty tuple
│ 
│ The given key does not identify an element in this collection value.
╵
```

there is no condition for `var.enable_ephemeral_storage`

_Originally posted by @borissavelev in https://github.com/terraform-aws-modules/terraform-aws-atlantis/pull/247#discussion_r806054799_

**Comment by MarkIannucci** [comment]
I'll look to correct this problem Thursday or Friday this week.

**Comment by bryantbiggs** [comment]
duplicate of #252

**Comment by borissavelev** [comment]
@bryantbiggs this one is not related to #252
And fix from #252 will not fix this issue

**Comment by bryantbiggs** [comment]
ah, I see - this is not conditionally blocked by a `var.enable_ephemeral_storage` https://github.com/terraform-aws-modules/terraform-aws-atlantis/blob/2df6bd6d5350d38e42ad9c63103c642f60293a43/main.tf#L436

**Comment by bryantbiggs** [comment]
whoever picks up this issue - we'll want to create a new example that has EFS enabled, and the current `complete-github` we can leave out EFS. Something like `efs-github` I guess

**Comment by antonbabenko** [comment]
It is so good to see you guys participating in this feature/bug resolution so I have made my first ever Terraform meme - https://twitter.com/antonbabenko/status/1493506138960273408

Thank you! 🚀

**Comment by jeff-clearcover** [comment]
@MarkIannucci did you pick this up?

**Comment by MarkIannucci** [comment]
@jeff-clearcover , yep.  I've got some code which fixes this underway, I'll push and open a PR later tonight (after dinner with my family :-)

I am hoping to increase the example coverage a bit (adding one for VPC and a second for ephemeral storage)

**Comment by jeff-clearcover** [comment]
Woohoo, you rock @MarkIannucci thank you for the quick reply ... here to test and help as needed too!

**Comment by antonbabenko** [comment]
This issue has been resolved in version 3.13.1 :tada:
